src/common/const.ts

```diff
@@ -254,3 +254,26 @@ export const UNIT_F = "°F";
 
 /** Entity ID of the default view. */
 export const DEFAULT_VIEW_ENTITY_ID = "group.default_view";
+
+/** State attributes which should not be shown in more-info window */
+export const LOGICAL_STATE_ATTRIBUTES = [
```

Comment:
Let's not move it inside `const.ts` because I don't think that code splitting is that good. It means this variable is loaded on every laod that uses something from `const.ts`.

We should break up this file and not add more stuff. 

Can you make a file `src/data/entity_attributes.ts` maybe?